### ISSUE TITLE
[Feature Request] Android Auto support

### ISSUE THREAD

**Issue by gbakeman** [zyrouge/symphony#45]
I'd love to see an Android Auto UI with support for playing, pausing and skipping, as well as resuming where I left off.

Thanks for making this!

**Comment by zyrouge** [comment]
Related article: https://developer.android.com/training/cars/media/auto.

**Comment by danielloera** [comment]
Hey @zyrouge I looked into the requirements for this and I got a bit confused on this part:
https://developer.android.com/training/cars/media#onGetRoot
Namely, the part where we have to provider the browser root
 i.e. `MediaBrowserServiceCompat.BrowserRoot(MY_MEDIA_ROOT_ID, null)`
 
 I would _assume_ that we could gather this info from Symphony.kt, but from my understanding there isn't an easy way to share ViewModel data with a service. Am I missing something here? Apologies in advance if this is a dumb question, its been a while since I worked on Android things.

**Comment by zyrouge** [comment]
> Hey @zyrouge I looked into the requirements for this and I got a bit confused on this part: https://developer.android.com/training/cars/media#onGetRoot Namely, the part where we have to provide the browser root i.e. `MediaBrowserServiceCompat.BrowserRoot(MY_MEDIA_ROOT_ID, null)`
> 
> I would _assume_ that we could gather this info from Symphony.kt, but from my understanding there isn't an easy way to share ViewModel data with a service. Am I missing something here? Apologies in advance if this is a dumb question, its been a while since I worked on Android things.

True, ViewModel cannot be easily shared. It seems like it requires only `MEDIA_ROOT_ID`, so try getting it. I'm not sure how services actually work, so I couldn't help much. If you are working on this, open a draft PR and I'll try to help you out there.

**Comment by CubeOfCheese** [comment]
Since Symphony is only available on F-Droid, Android auto support isn't really possible. 

https://github.com/ev-map/EVMap/issues/319#issuecomment-1880029130

Basically Android Auto only permits apps installed through Google play unfortunately. Too bad Android Auto isn't open source.

**Comment by zyrouge** [comment]
Oh, I wasn't aware of it. Maybe we'll implement something like car mode where the controls are larger. Nonetheless, this needs to be discussed more.

**Comment by dephunk** [comment]
Since symphony is now on the Play store are you looking into Android auto? It's the only feature keeping me from using it as my daily. Otherwise love the app.

**Comment by MaezrFulldive** [comment]
Android auto support is available to fdroid apps, but developer mode or third party software must be enabled in the android auto settings.

**Comment by dephunk** [comment]
> Android auto support is available to fdroid apps, but developer mode or third party software must be enabled in the android auto settings.

Android auto is not working with either the Play store or f droid app, even with dev mode + 3rd party software enabled.

**Comment by opluan** [comment]
Any updates on this request? I'm loving this app, but now i need it to run at android auto =(
